os/kernel/binary_manager/binary_manager_data.c

```diff
@@ -145,11 +149,25 @@ bool binary_manager_scan_kbin(void)
 int binary_manager_check_kernel_update(void)
 {
 	int ret;
+	int inactive_partidx;
 	char filepath[BINARY_PATH_LEN];
 	kernel_binary_header_t header_data;
 
+	inactive_partidx = kernel_info.inuse_idx ^ 1;
+
+#ifdef CONFIG_BINARY_SIGNING
+	/* Check signature */
+	ret = up_verify_kernelsignature(kernel_info.part_info[inactive_partidx].address);
+	if (ret == 1) {
+		bmvdbg("Kernel Signature Checking Success\n");
```

Comment:
Sometimes, you print the signature result log with `printf`, but sometimes with `dbg`.
Is there any reason that you distinguish the logs?

Reply:
I used `printf` for logs in booting or loading time.
So I used `bmXdbg` here because this function is called to check whether new valid binary exist or not when `binary_manager_set_bootparam`, not loading time.